Sources/Internal/UpdateSessionHandler.swift

```diff
@@ -32,4 +32,6 @@ class UpdateSessionHandler: RequestHandler {
 
 struct SessionInfo: Decodable {
     var approved: Bool
+    var accounts: [String]?
+    var chainId: Int
```

Comment:
@pedrouid
1. Does it make sense to make `chainId` optional for wc_sessionUpdate?
2. How to treat empty or absent accounts list if `approved=true`? My guess is that if `accounts` are not passed, then the old array is not changed. Should we send an error response if `approved=true` and `accounts=[]`?

Reply:
@shmakovigor do you mind if we make chainId optional here?